.travis.yml

```diff
@@ -7,9 +7,17 @@ php:
 
 before_script:
   - wget http://getcomposer.org/composer.phar
+  - if [[ "$PHP" > "5.3" ]]; then composer require satooshi/php-coveralls:dev-master; fi
   - php composer.phar install --no-interaction
   - php composer.phar dump-autoload --optimize
-  
+
+script:
+  - mkdir ./tests/logs/
+  - ./vendor/bin/phpunit --coverage-clover ./tests/logs/clover.xml
+
+after_script:
+  - travis_retry php vendor/bin/coveralls -v
```

Comment:
good point @tompedals - for the `phpunit` call it shouldn't matter... but if travis fails and needs to retry then yes it'll try and run a binary that doesn't exist so will modify the `after_script` block